src/phasorpy/phasor.py

```diff
@@ -992,10 +993,16 @@ def phasor_calibrate(
         Must be measured with the same instrument setting as the phasor
         coordinates to be calibrated.
     frequency : array_like
-        Laser pulse or modulation frequency in MHz.
-        A scalar or one-dimensional sequence.
+        Fundamental laser pulse or modulation frequency in MHz.
     lifetime : array_like
         Lifetime components in ns. Must be scalar or one dimensional.
+    harmonic : array_like, optional
+        Harmonics used for calibration.
+    skip_axis : int or sequence of int, optional
+        Axes to be excluded during center calculation. If None, all
+        axes are considered. Axes with harmonics should be excluded.
+        If multiple harmonics are used, axes should be specified, else
+        the first axis is skipped by default.
```

Comment:
@cgohlke do you think the `skip_axis` parameter name should be changed to something more explicitly linked to the harmonics? Also, I think the description of the parameter needs improving.

Reply:
I would leave `skip_axis` and `frequency` as is. It should still work to calibrate mulit-harmonic coordinates using just those two parameters. The `harmonic` parameter is added for pure convenience, modifying `skip_axis` and `frequency` internally, raising exceptions if incompatible. That should be straightforward, or am I missing something?